<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>フォント</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>フォントサイズ:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="fontSizeBox">
           <property name="minimum">
            <double>8.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>行間:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="lineHeightBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>色</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="fontColorCodeLine"/>
          </item>
          <item>
           <widget class="QPushButton" name="selectFontColorButton">
            <property name="text">
             <string>フォントカラーの選択</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>背景</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>背景色</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="transparentBackgroundBox">
            <property name="text">
             <string>透明</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="backgroundColorCodeLine"/>
          </item>
          <item>
           <widget class="QPushButton" name="selectBackgroundColorButton">
            <property name="text">
             <string>背景色を選択する</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>マージン</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>上:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="topMarginBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>px</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>下:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="bottomMarginBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>px</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>左:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="leftMarginBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>px</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>右:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="rightMarginBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>px</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>ファイル名</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>ファイル名</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>拡張子は入力しない。</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fileNameLine"/>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>実際のファイル名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="realFileNameLine">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>語群</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="1">
           <widget class="QPushButton" name="createdHourButton">
            <property name="text">
             <string>作成時</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="createdMinuteButton">
            <property name="text">
             <string>作成分</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="createdDayButton">
            <property name="text">
             <string>作成日</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="createdSecondButton">
            <property name="text">
             <string>作成秒</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="createdMonthButton">
            <property name="text">
             <string>作成月</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>クリックすると入力されます。</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="createdYearButton">
            <property name="text">
             <string>作成年</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="createdUnixTimeButton">
            <property name="text">
             <string>UNIX time</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>transparentBackgroundBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>backgroundColorCodeLine</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transparentBackgroundBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectBackgroundColorButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
